pkg/proto/configuration/global/global.proto

```diff
@@ -68,6 +88,9 @@ message TracingConfiguration {
 
   // Whether or not all traces should be sampled.
   bool always_sample = 4;
+
+  // Policy to use to decide to sample traces.
+  SamplingPolicy sampling_policy = 5;
```

Comment:
As it is right now, the schema adds some ambiguity. There is no requirement that a `oneof` is set, meaning that `samplingPolicy: {}` is also valid, schema-wise. It means there is a SamplingPolicy set, but that none of the children are set. What are your thoughts on simply doing something like this?

```proto
message TracingConfiguration {
  ...
  oneof sample_policy {
    google.protobuf.Empty sample_always = 5;
    google.protobuf.Empty sample_never = 6;
    double sample_probability = 7;
  }
}
```

Reply:
That suggestion is fine. Will switch it.